scripts/test-check-packages.sh

```diff
@@ -85,10 +85,9 @@ for d in test/packages/${PACKAGE_TEST_TYPE:-other}/${PACKAGE_UNDER_TEST:-*}/; do
           --old ${OLDPWD}/build/benchmark-results-old \
           --threshold 1 --report-output-path="${OLDPWD}/build/benchreport"
       fi
-      # FIXME: running system benchmark in package "system_benchmark" fails with panic
-      # if [ "${package_to_test}" == "system_benchmark" ]; then
-      #   elastic-package benchmark system --benchmark logs-benchmark -v --defer-cleanup 1s
-      # fi
+      if [ "${package_to_test}" == "system_benchmark" ]; then
+        elastic-package benchmark system --benchmark logs-benchmark -v --defer-cleanup 1s
+      fi
```

Comment:
Nice ! 
I think it can be removed from this command the `--defer-cleanup` parameter. Maybe , is it just there because of copying from the test command ?

```suggestion
      if [ "${package_to_test}" == "system_benchmark" ]; then
        elastic-package benchmark system --benchmark logs-benchmark -v
      fi
```

Reply:
It is there on purpose. I think it might be good to debug the benchmark setup in some circumstances since it let you prevent tearing down services and cleaning up the data.